src/openlifu/db/database.py

```diff
@@ -575,6 +626,13 @@ def get_volume_dir(self, subject_id, volume_id):
     def get_volume_metadata_filepath(self, subject_id, volume_id):
         return Path(self.get_volume_dir(subject_id, volume_id)) / f'{volume_id}.json'
 
+    def get_run_dir(self, subject_id, session_id, run_id):
+        run_dir = self.get_session_dir(subject_id, session_id) / 'runs' / f'{run_id}'
+        return run_dir
+
+    def get_run_metadata_filepath(self, subject_id, session_id, run_id):
```

Comment:
(in the case of `Run`s it's not just metatdata but rather the full definition)

```suggestion
    def get_run_filepath(self, subject_id, session_id, run_id):
```